test/system/700-play.bats

```diff
@@ -981,3 +981,45 @@ _EOF
     run_podman pod rm -t 0 -f test_pod
     run_podman rmi -f userimage:latest $from_image
 }
+
+@test "podman play with automount volume" {
```

Comment:
@edsantiago PTAL if you get a chance, this test works but it can probably be improved